lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -1760,6 +1760,166 @@ LogicalResult ConvertAtenOp<AtenConv2dOp>::matchAndRewrite(
   return success();
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenReshapeOp>::matchAndRewrite(
+    AtenReshapeOp op, OpAdaptor adaptor,
+    ConversionPatternRewriter &rewriter) const {
+
+  auto self = adaptor.self();
+
+  auto selfTy = self.getType().template cast<RankedTensorType>();
+  if (!selfTy)
+    return op.emitError("Only ranked tensor types supported in TOSA Reshape");
+
+  // Check that at most one dimension is -1
+  SmallVector<int64_t> newShape;
+  if (!matchPattern(op.shape(), m_TorchConstantIntList(newShape)))
+    return op.emitError("Only constant shape supported in TOSA Reshape");
+
+  int auto_sz = 0;
+  for (auto s : newShape)
+    auto_sz += (s == -1 ? 1 : 0);
+  if (auto_sz > 1)
+    op.emitError("At most one dimension may be specified as -1 to "
+                 "automatically calculate its size");
+
+  auto newType = RankedTensorType::get(newShape, selfTy.getElementType());
+
+  rewriter.replaceOpWithNewOp<tosa::ReshapeOp>(
+      op, getTypeConverter()->convertType(newType), self,
+      rewriter.getI64ArrayAttr(newShape));
+
+  return success();
+}
+
+// Normalization ops perform elementwise ops of a single mean/stdev value
+// against the feature map and because input is NCHW, the rank-1 value must be
+// reshaped so it sits on the same dim as 'C'.
+static LogicalResult reshapeToNormInputDim(Operation *op,
+                                           ConversionPatternRewriter &rewriter,
+                                           TypeConverter *converter,
+                                           Type outType, const Value &toBcast,
```

Comment:
nit: don't need & when passing Value type. Value type itself is already a pointer wrapper of the storage type.